models/appModels.js

```diff
@@ -80,25 +80,44 @@ if (!greenlistedSortBys.includes(sort_by) || !greenlistedOrders.includes(order))
 }
 
 const fetchSpecificArticle = (id) => {
-    const queryString = 'SELECT * FROM articles WHERE article_id = $1;'
-    const queryValues = [];
-
-    //The below code works but a simpler way is to simply let the ID go through if it is not numbers - this will trigger and SQL error which can simply be handled in our error handlers
-    // if (typeof +id !== 'number' || isNaN(+id)){
-    //     return Promise.reject({status : 400, msg: 'Bad Request'})
+    // const queryString = 'SELECT * FROM articles WHERE article_id = $1;'
```

Comment:
be wary of leaving in commented and redundant code especially when you are pushing to GitHub